vtex-apps/admin-pages
# NAD-530 - Add temporary newadmin warning

#### What problem is this solving?

Warns customers on the `newadmin` production workspaces, which hosts the Admin's beta version, to navigate back to the master workspace in case they want to edit their stores.

#### How should this be manually tested?

Navigate to this [workspace](https://newadmin--teamadmin.myvtex.com/admin/cms/site-editor) and make sure that you see an alert on top of the Site Editor 😉 

NOTE: Although you'll see the yellow, development banner, this won't be true once this PR is merged, because the linked version doesn't contain the `production` check.

NOTE 2: These changes should only be apparent for customers on the `admin@4.x`, which doesn't have the same amount of languages available as the `admin@3.x`, thus, there are English translations placed on other languages, as the CI will fail in case the `intl-equalizer` doesn't find the same translation keys in all files.

#### Type of changes



| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| \_  | Bug fix                                |
| ✔️   | New feature                        |
| \_  | Breaking change  |
| \_  | Technical improvements  |

#### Notes

For more context on these changes, please reach #team-admin on Slack.
